### ISSUE TITLE
Fix npm dependency vulnerabilities

### ISSUE THREAD

**Issue by IoPhoenix** [hatchways/team-evergreen#187]
- Fix errors in `npm audit`

**Comment by filg68** [comment]
Hey didn't you fix this already?

**Comment by IoPhoenix** [comment]
No, there seems to be an issue with the parent package and there is no patch for this error yet :(

**Comment by filg68** [comment]
There's not much we can do with package dependencies except wait until its fixed at source or find something else in its place.  In this case as the package in question is npm I think we need to wait.
